resotolib/resotolib/baseresources.py

```diff
@@ -854,6 +854,12 @@ class BaseBucket(BaseResource):
     kind: ClassVar[str] = "bucket"
 
 
+@define(eq=False, slots=False)
+class BaseServerlessFunction(BaseResource):
+    kind: ClassVar[str] = "serverless_function"
+    fingerprint: str = ""
```

Comment:
Just stumble upon this property, which is not filled in the lambda collector. Why is this?